test/test_unary_ufuncs.py

```diff
@@ -469,30 +468,6 @@ def test_batch_vs_slicing(self, device, dtype, op):
 
         self.assertEqual(actual, expected)
 
-    def _test_out_arg(self, op, input, output, expected):
-        if op.safe_casts_outputs:
-            expect_fail = not torch.can_cast(expected.dtype, output.dtype)
-        else:
-            expect_fail = output.dtype != expected.dtype
-
-        if expect_fail:
-            with self.assertRaises(RuntimeError):
-                op(input, out=output)
-        else:
-            res = op(input, out=output)
-            self.assertTrue(res is output)
-            self.assertEqual(output, expected.to(output.dtype))
-
-    @ops(unary_ufuncs, dtypes=OpDTypes.supported)
-    def test_out_arg_all_dtypes(self, device, dtype, op):
```

Comment:
it looks like functionality of this test is not tested in the new out testing?

Reply:
Yep; added back.